src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx

```diff
@@ -230,31 +227,25 @@ During this phase, you will introduce incident management teams and other stakeh
 
 The process consists of four activities:
 
-1.  Review AQM Dashboard and KPI Trending - Here you and the stakeholders will look at the AQM KPIs and identify their week over week trends.  The team should identify areas where KPIs are not improving and develop strategies to drive improvement.
-
-2.  Identify Achievements, Challenges, and Opportunities - Here you and the stakeholders will map the current state of alert quality to business impact, identifying areas where improvement has resulted in better business outcomes and areas where problems are impacting business outcomes.
-
-3.  Incident Policy Review - Using the AQM dashboard, you and the stakeholders will identify the noisiest incident policies.  Once identified, those policies should be evaluated as detailed in step 4 below.
-
-4.  Alert Policy Recommendations - In this step, you and the stakeholders will review the noisiest policies using the following criteria:
-
-- Do the alerts have any business impact?
-- Are the policies properly configured?
-  - Are they telling us something about the resource that needs to be fixed?
-  - Are the policies necessary? (i.e. do they have business impact)
-  - Are the thresholds set properly?
-
-5.  Technical recommendations - Here, you and the stakeholders will review any technical recommendations, including: 
-
-- Are there application / system problems for engineering to review?
-- Are there poorly constructed policies that need to be fixed?
-- Are there instrumentation gaps?
+1. Review AQM Dashboard and KPI Trending: Here you and the stakeholders will look at the AQM KPIs and identify their week over week trends.  The team should identify areas where KPIs are not improving and develop strategies to drive improvement.
+2. Identify Achievements, Challenges, and Opportunities: Here you and the stakeholders will map the current state of alert quality to business impact, identifying areas where improvement has resulted in better business outcomes and areas where problems are impacting business outcomes.
+3. Incident Policy Review: Using the AQM dashboard, you and the stakeholders will identify the noisiest incident policies.  Once identified, those policies should be evaluated as detailed in step 4 below.
+4. Alert Policy Recommendations: In this step, you and the stakeholders will review the noisiest policies using the following criteria:
+    - Do the alerts have any business impact?
```

Comment:
@austin-schaefer, should we remove the blank line separating this child bullet from its sibling? This extra line doesn't seem like the pattern we've been using.

Reply:
Hey @rhetoric101 , not seeing a blank line in my editor?

![image](https://user-images.githubusercontent.com/55203603/141372879-6a92c64c-1f5a-437d-915c-577b0fd00216.png)